src/App.tsx

```diff
@@ -1,12 +1,69 @@
-import { Box, Typography } from "decentraland-ui2"
+import { useState } from "react"
+import { ThemeProvider, dark } from "decentraland-ui2/dist/theme"
+import {
+  Box,
+  CircularProgress,
+  CssBaseline,
+  Toolbar,
+  Typography,
+} from "@mui/material"
```

Comment:
```suggestion
} from "decentraland-ui2"
```

it's possible import the `@mui/material` components directly from `decentraland-ui2`